ControlRoomApplication/ControlRoomApplication/GUI/RTControlForm.cs

```diff
@@ -606,6 +610,20 @@ private void controlScriptsCombo_SelectedIndexChanged(object sender, EventArgs e
 
         }
 
+        /// <summary>
+        /// Functionality for when the STOP Telescope button is pressed. Creates a confirmation pop up box to prevent accidental presses of the stop button before sending any stop script.
+        /// </summary>
+        private void stopRT_click(object sender, EventArgs e)
+        {
+            DialogResult result = MessageBox.Show("Are you sure you want to stop the telescope?", "Telescope Stop Confirmation", MessageBoxButtons.YesNo);
+
+            if (result == DialogResult.Yes)
+            {
+                // Run the stop script for the telescope
+                rtController.RadioTelescope.PLCDriver.InterruptMovementAndWaitUntilStopped(true, false);
```

Comment:
Since the optional software stops parameter defaults to `false`, we can just remove the passed in `false` completely and the value will still be false.

- [ ] Remove redundant false paramter